### ISSUE TITLE
маркетинговый материал для розничной торговли

### ISSUE THREAD

**Issue by gsbelarus** [GoldenSoftwareLtd/gedemin#1462]
Originally reported on Google Code with ID 1462

```
Подготовить маркетинговый материал по программному комплексу "Гедымин" для
розничной торговли для сайта www.garant.by (раздел кассовые системы) и для
раздаточного материала в ЦТО, с целью популяризации продукта и увеличения
продаж в Гродненской области.

```

Reported by `sjuchniewicz` on 2009-07-06 14:29:28

**Comment by gsbelarus** [comment]
Reported by `sjuchniewicz` on 2009-07-06 14:30:08
- **Status changed**: `New`

**Comment by gsbelarus** [comment]
Reported by `sjuchniewicz` on 2009-07-06 14:30:59
- **Labels added**: Type-Task, Priority-Medium

**Comment by gsbelarus** [comment]
Reported by `gs1994` on 2009-07-09 14:36:41
- **Labels added**: Component-Docs

**Comment by gsbelarus** [comment]
Reported by `gs1994` on 2009-07-12 16:50:48
- **Labels added**: Cash

**Comment by gsbelarus** [comment]
Reported by `gs1994` on 2009-10-08 11:13:08
- **Labels added**: Garant

**Comment by gsbelarus** [comment]
Reported by `alexandra.gsoftware` on 2010-03-31 18:50:32
- **Status changed**: `Duplicate`
- **Merged into**: #1587
